packages/legend-extension-dsl-service/src/stores/studio/UpdateServiceQuerySetupStore.ts

```diff
@@ -112,12 +124,71 @@ export class UpdateServiceQuerySetupStore {
     this.applicationStore.navigator.goTo(
       generateServiceQueryUpdaterSetupRoute(undefined, undefined, undefined),
     );
+    this.resetProjectConfiguration();
   }
 
   resetCurrentGroupWorkspace(): void {
     this.currentGroupWorkspace = undefined;
     this.currentWorkspaceService = undefined;
   }
+  setIsCurrentProjectNotConfigured(val: boolean | undefined): void {
+    this.isCurrentProjectNotConfigured = val;
```

Comment:
with https://github.com/finos/legend-studio/pull/1506 doe, we should also consider what we should do there, people can't create new workspace so that flow is basically blocked for that case.